### PR TITLE
fix: remove broken symlink tests

### PR DESCRIPTION
**Pull request by vlasy**
Broken symlink causes github action to crash

## Description

Remove broken symlink tests. Broken symlink file causes github action checkout to fail

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/services/FileSystemService.real.spec.ts [hunk 1]
```diff
@@ -228,7 +228,7 @@ describe('FileSystemService - REAL', () => {
       await expect(fileSystemService.createDirectory(mockFilePath)).rejects.toThrow('EEXIST');
     });
 
-    it('throws an error if the target is a broken symbolc link', async () => {
+    it.skip('throws an error if the target is a broken symbolc link', async () => {
```
Comment (adelkahomolova) [inline src/services/FileSystemService.real.spec.ts:231]: Add comment whit it is skipped.

File: src/services/FileSystemService.real.spec.ts [hunk 2]
```diff
@@ -295,7 +295,7 @@ describe('FileSystemService - REAL', () => {
         expect(result.type).toEqual('file');
       });
 
-      it('it returns metadata for broken Symlink but only as a File', async () => {
+      it.skip('it returns metadata for broken Symlink but only as a File', async () => {
```
Comment (adelkahomolova) [inline src/services/FileSystemService.real.spec.ts:298]: Add comment whit it is skipped.

**Review by adelkahomolova — CHANGES_REQUESTED** [review]

File: src/services/FileSystemService.real.spec.ts (hunk 1 above)
Comment (vlasy) [reply src/services/FileSystemService.real.spec.ts:232]: Fixed in b4c96e4c7244afcd90bac9258a1241bb26cc4304

File: src/services/FileSystemService.real.spec.ts (hunk 2 above)
Comment (vlasy) [reply src/services/FileSystemService.real.spec.ts:300]: Fixed in b4c96e4c7244afcd90bac9258a1241bb26cc4304

**Review by adelkahomolova — APPROVED** [review]

**Pull request merged**